doc/whats_new/v0.20.rst

```diff
@@ -120,6 +120,11 @@ Classifiers and regressors
   error for prior list which summed to 1.
   :issue:`10005` by :user:`Gaurav Dhingra <gxyd>`.
 
+- Fixed a bug in :class:`linear_model.LogisticRegression` where when using the
+  parameter ``multi_class='multinomial'``, the ``predict_proba`` method was
+  returning incorrect probabilities in the case of binary outcomes.
+  :issue:`9939` by :user: `rwolst`.
```

Comment:
No space between colon and backtick please. You're also welcome to include your real name

Reply:
@rwolst you can find an example of the syntax to use in the previous entry.

Reply:
Ok, will fix this in next commit.